kyo-combinators/shared/src/main/scala/kyo/Constructors.scala

```diff
@@ -89,8 +86,8 @@ extension (kyoObject: Kyo.type)
       * @return
       *   An effect that prints the message to the console
       */
-    def debugln[S](message: => String < S)(using Frame): Unit < (S & IO) =
-        message.map(m => Console.println(m))
+    def debugln[S](message: String)(using Frame): Unit < IO =
```

Comment:
This one seems less important to remove effectual parameter... Regardless there is a leftover `[S]`